pyronear/models/densenet.py

```diff
@@ -0,0 +1,150 @@
+#!usr/bin/python
+# -*- coding: utf-8 -*-
+
+import re
+from torchvision.models.densenet import DenseNet, model_urls as imagenet_urls
+from torchvision.models.utils import load_state_dict_from_url
+from .utils import cnn_model
+
+__all__ = ['densenet121', 'densenet169', 'densenet201', 'densenet161']
+
+
+model_urls = {
+    'densenet121': 'https://srv-file7.gofile.io/download/XqHLBB/densenet121-binary-classification.pth'
```

Comment:
Is it normal that there is only a URL for densenet121 and not the others ?

Reply:
It's just that I didn't have time to train the others!
But the same goes for all `resnext` and `wide_resnet` unfortunately. I thought we would deal with that in the next release (since it takes at least 40-45min per model and there are quite a lot of them), what do you think?

Reply:
If it does not affect the normal usage and the user will get an understandable error when trying to load a pretrained model (if I understood the idea), I think it should be fine